gabarit/template_vision/vision_project/package_name/monitoring/model_explainer.py

```diff
@@ -161,6 +170,11 @@ def explain_instance_as_html(self, *args, **kwargs):
         '''
         raise NotImplementedError("'explain_instance_as_html' is not defined for LimeExplainer")
 
+    def explain_instance_as_json(self, *args, **kwargs):
```

Comment:
Isn't it better to add the same arguments as the parent class ? If so, we must do the same for the other methods.